src/dev/impl/DevToys/Views/Tools/Converters/Timestamp/TimestampToolPage.xaml

```diff
@@ -9,41 +9,206 @@
     mc:Ignorable="d"
     d:DesignHeight="400"
     d:DesignWidth="600">
-    <StackPanel Margin="0,0,0,16" Spacing="12">
+
+    <StackPanel
+        Margin="0,0,0,16"
+        Spacing="12">
+
+        <VisualStateManager.VisualStateGroups>
+            <VisualStateGroup>
+                <VisualState>
+                    <VisualState.StateTriggers>
+                        <AdaptiveTrigger
+                            MinWindowWidth="640" />
+                    </VisualState.StateTriggers>
+
+                    <VisualState.Setters>
+                        <Setter
+                            Target="CurrentDateGridBlock.Grid.ColumnSpan"
+                            Value="1" />
+
+                        <Setter
+                            Target="CurrentTimeGridBlock.Grid.Row"
+                            Value="0" />
+                        <Setter
+                            Target="CurrentTimeGridBlock.Grid.Column"
+                            Value="1" />
+                        <Setter
+                            Target="CurrentTimeGridBlock.Grid.ColumnSpan"
+                            Value="1" />
+                    </VisualState.Setters>
+                </VisualState>
+            </VisualStateGroup>
+        </VisualStateManager.VisualStateGroups>
+
+
         <StackPanel.ChildrenTransitions>
             <TransitionCollection>
-                <EntranceThemeTransition IsStaggeringEnabled="True" FromVerticalOffset="50"/>
+                <EntranceThemeTransition
+                    IsStaggeringEnabled="True"
+                    FromVerticalOffset="50" />
             </TransitionCollection>
         </StackPanel.ChildrenTransitions>
 
-        <StackPanel Margin="0,12,0,0">
-
+        <StackPanel
+            Margin="0,12,0,0">
             <muxc:InfoBar
```

Comment:
Silly question: does this info bar still get displayed at some point? I tried to play with all the fields to "break" it, but at the end it seems like the timestamp is being fixed by itself, which makes that I don't succeed to get this info bar to be displayed, which makes me think that perhaps we can remove it?